README.md

```diff
@@ -15,61 +15,86 @@ See [documentation](https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en
 Documentation
 -------------
 
-`Text Processing (text normalization and inverse text normalization) <https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/main/nlp/text_normalization/intro.html>`_
+[Text Processing (text normalization and inverse text normalization)](https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/main/nlp/text_normalization/intro.html).
 
 Tutorials
 ---------
-A great way to start with NeMo is by checking `one of our tutorials <https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/stable/starthere/tutorials.html>`_.
+A great way to start with NeMo is by checking [one of our tutorials](https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/stable/starthere/tutorials.html).
```

Comment:
"to start with NeMo" - NeMo -> NeMo-TN. One 2 tutorials are relevant to the current repo and they both are hosted here. How about creating a table in the README?

Reply:
this is not resolved - could we list 2 relevant tutorials in the readme or point to https://github.com/NVIDIA/NeMo-text-processing/tree/main/tutorials ?